.github/issue_template.md

```diff
@@ -0,0 +1,31 @@
+<!-- Thanks for taking the time to file an issue. Someone from our team will take a look and get back to you as soon as possible. -->
+
+## Issue Type
+<!-- Please check one of the following with an “x” to indicate what type of issue you’re submitting-->
+[ ] Bug Report <!-- Please search GitHub for a similar issue or PR before submitting -->  
+[ ] Feature Request <!-- Please file a UserVoice request and include the link below https://wpdev.uservoice.com/forums/110705-universal-windows-platform/category/58517-xaml-controls-composition -->  
+[ ] Sample request  
+[ ] Documentation issue or request  
+[ ] Other – Please describe:   
+
+## UserVoice link
+<!-- If requesting new functionality -->
+
+## Current Behavior
+<!-- If filing a bug, describe what happens instead of the expected behavior. Please include a screenshot or gif if applicable. -->
+<!-- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Expected Behavior
+<!-- If filing a bug, tell us what expected/desired behavior is -->
+<!-- If suggesting a change/improvement, tell us how it should work -->
+
+## Steps to Reproduce (for bugs)
+<!-- Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include code to reproduce, if relevant -->
+1.  
+2.  
+3.  
+
+##  Your Environment
+<!-- Provide details on relevant environment version numbers -->
+* SDK #:  
+* Operating System build # :  
```

Comment:
Maybe Visual Studio version number would also be useful?

Reply:
Updated